src/api-v1.js

```diff
@@ -779,51 +735,3 @@ api.declare({
     uptime: process.uptime(),
   });
 });
-
-api.declare({
-  method: 'get',
-  route: '/backend-status',
```

Comment:
It's not -- status comes from the ping endpoint.  I'll file a bug to document this endpoint and its schemas, and to update the status tool (not status.tc.n) to use that endpoint